Detectors/ITSMFT/ITS/reconstruction/include/ITSReconstruction/Cluster.h

```diff
@@ -54,10 +54,10 @@ class Cluster : public FairTimeStamp
     kMaskClUse = 0x7f
   };
 //
-#ifdef _ClusterTopology_
+//#ifdef _ClusterTopology_
```

Comment:
This reverts what has been introduced in commit 229c4db35c4d8c8fb05c6049fd005192f423d986 so the best is to squash those two commits